kratos/utilities/divide_triangle_2d_3.cpp

```diff
@@ -341,4 +334,17 @@ namespace Kratos
         }
     };
 
+    DivideTriangle2D3::IndexedPointGeometryPointerType DivideTriangle2D3::GenerateTriangle(const int& r_i0,const int& r_i1,const int& r_i2) {
+        return Kratos::make_shared<IndexedPointTriangleType>(
+                    mAuxPointsContainer(r_i0),
+                    mAuxPointsContainer(r_i1),
+                    mAuxPointsContainer(r_i2));
+    }
+
+    DivideTriangle2D3::IndexedPointGeometryPointerType DivideTriangle2D3::GenerateLine(const int& r_i0,const int& r_i1) {
+        return Kratos::make_shared<IndexedPointLineType>(
+                    mAuxPointsContainer(r_i0),
+                    mAuxPointsContainer(r_i1));
+    }
```

Comment:
```suggestion
    DivideTriangle2D3::IndexedPointGeometryPointerType DivideTriangle2D3::GenerateAuxiliaryPartitionTriangle(
        const int I0,
        const int I1
        const int I2)
    {
        return Kratos::make_shared<IndexedPointTriangleType>(
            mAuxPointsContainer(I0),
            mAuxPointsContainer(I1),
            mAuxPointsContainer(I2));
    }

    DivideTriangle2D3::IndexedPointGeometryPointerType DivideTriangle2D3::GenerateIntersectionLine(
        const int I0,
        const int I1)
    {
        return Kratos::make_shared<IndexedPointLineType>(
            mAuxPointsContainer(I0),
            mAuxPointsContainer(I1));
    }
```
Please note:
- the [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide) suggestions and the indentation (in Kratos we use 4 spaces)
- passing the arguments as reference for simple types makes no difference